CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Overview
+
+This documents explains the processes and practices recommended for
+contributing enhancements to the Tempo bundle.
+
+- Generally, before developing enhancements to this charm, you should consider
+  [opening an issue](https://github.com/canonical/tempo-bundle) explaining
+  your use case.
+- If you would like to chat with us about your use cases or proposed
+  implementation, you can reach us on the
+  [Charmhub Mattermost](https://chat.charmhub.io/charmhub/channels/charm-dev)
+  or [Discourse](https://discourse.charmhub.io/).
+- All enhancements require review before being merged.
+  Apart from code quality and test coverage, the review will also take into
+  account the Juju administrator user experience using the bundle.
+
+## Development
+
+There are two charms used for development:
+- tempo
+- s3proxy
+
+### Deploy with local charms
+
+To deploy the bundle using only/some local charms you need to render the
+[`bundle.yaml.j2`](bundle.yaml.j2) template and then deploy the rendered bundle
```

Comment:
mmm this is not correct, is it? By default the bundle points to charmhub charms so it will deploy tempo from stable/edge...? IIRC you need to go and change the charm name in the rendered bundle to point it to a local path for this to work.